pages/managed-mongodb-databases/how-to/create-a-database-instance.mdx

```diff
@@ -38,16 +38,32 @@ Document databases enable users to store and retrieve data in a document format,
         - **Replica-set 1-node** - an instance of MongoDB® that runs as a single server and does not provide redundancy or high availability.
         - **Replica-set 3-nodes** - a group of 3 MongoDB® servers (1 primary and 2 standby nodes) that maintain the same data set. Replica sets provide redundancy and high availability and are the basis for all production deployments. If the main node fails for any reason, one of the remaining standby nodes is assigned and can take over requests, reducing downtime.
     - Select a node type.
-    - Define the size of your Block Storage volume. You can increase your storage space without changing your node type, with no downtime. You can increase your volume size to up to 10 TB.
-    - Add a name and set a password for your user.
-    <Message type="important">
-    Your username must adhere to specific criteria.
-      - Length must be between 1 and 63 characters
-      - First character must be an alphabetic character (**a-Za-Z**)
-      - It can not start with **_rdb**
-      - Only **a-zA-Z0-9_$-** characters are accepted
-    </Message>
+    - Configure storage settings. You can:
+      - Choose the [IOPS](/block-storage/concepts/#iops) setting, between **5k** and **15k**.
+      - Define your volume size. Block Storage volumes can increase up to 10 TB.
+    - Review the snapshot configuration. During the private beta, only manual snapshots are available.
+4. Configure your Network. Choose one or both of the options below:
+    - **Attach to Private Network**: allows your databases to communicate in an isolated and secure network without requiring a public IP address. If you select this option, you must either:
+      - Select and attach an existing Private Network and select the network from the drop-down list.
+          <Message type="important">
+            The Database Instance must be attached to a Private Network in the same region.
+          </Message>
+      - Create and attach a new Private Network and enter the name of the network in the form.
+          <Message type="important">
+            If you select this option, a Private Network with default settings will be created in the same region as your Database Instance. Do not use a TLD (e.g. dev, cloud) as a name, to avoid conflicts.
```

Comment:
```suggestion
            If you select this option, a Private Network with default settings will be created in the same region as your Database Instance. Do not use a TLD (e.g. `dev` or `cloud`) as a name, to avoid conflicts.
```